aggregator/entry_test.go

```diff
@@ -807,9 +802,13 @@ func testEntryAddMetricWithPoliciesList(
 				id := elem.Value.(*CounterElem).ID()
 				require.Equal(t, testCounterID, id)
 				aggregations := elem.Value.(*CounterElem).values
-				require.Equal(t, 1, len(aggregations))
-				require.Equal(t, alignedStart.UnixNano(), aggregations[0].timeNanos)
-				require.Equal(t, int64(1234), aggregations[0].counter.Sum())
+				if !expectedShouldAdd {
```

Comment:
nit: 
`if expectedShouldAdd {...} else {...}`

Reply:
Meh I usually put the shorter block first to make it easier on the eyes.